Mathlib/Analysis/Normed/Order/UpperLower.lean

```diff
@@ -154,3 +206,63 @@ theorem IsLowerSet.exists_subset_ball (hs : IsLowerSet s) (hx : x ∈ closure s)
 #align is_lower_set.exists_subset_ball IsLowerSet.exists_subset_ball
 
 end Fintype
+
```

Comment:
What happened to `is_antichain.interior_eq_empty`?

Reply:
It has already been ported (and generalised) as `IsAntichain.interior_eq_empty`